TeslaSUI/TeslaSUI/Views/ClimateView.swift

```diff
@@ -0,0 +1,393 @@
+                        backButtonView
+                        Spacer()
+                        screenNameTextView
+                        Spacer()
+                        settingsButtonView
+                    }
+                    ZStack {
+                        progressView
+                        trimCirlceProgress
+                        if climateViewModel.isAlertShown {
+                            alertView
+                        } else {
+                            EmptyView()
+                        }
+                    }
+                    Spacer()
+                    climateSettingsView
+                    Spacer()
+                }
+                BottomSheetView {
+                    VStack {
+                        bottomSheetCapsuleView
+                        Spacer()
+                        HStack {
+                            bottomSheetTextView
+                            Spacer()
+                            acIsOnButtonView
+                        }
+                        HStack(spacing: -20) {
+                            colorPickerView
+                            stepperView
+                            ventButtonView
+                        }
+                        .padding(.top)
+                        Spacer(minLength: 240)
+                    }
+                }
+            }
+        }
+    }
+    
+    // MARK: Private Propeties
+    
+    @StateObject private var climateViewModel = ClimateViewModel()
+    
+    private var gradient: LinearGradient {
+        LinearGradient(colors: [.topGradient, .bottomGradient],
+                       startPoint: .bottom,
+                       endPoint: .top)
+    }
+    
+    private var alertView: some View {
+        VStack {
+            if let url = URL(string: Constants.Text.teslaSupportLink) {
+                Link(destination: url) {
+                    Image(systemName: Constants.Images.linkImageName)
+                        .resizable()
+                        .frame(width: 50, height: 50)
+                }
+                Button(Constants.Text.closeButtonLabel) {
+                    withAnimation(.linear) {
+                        climateViewModel.isAlertShown.toggle()
+                    }
+                }
+                .foregroundColor(.white)
+            }
+        }
+        .background(Circle()
+            .fill(Color.backgrounElement)
+            .frame(width: 100, height: 100)
+        )
+    }
+    
+    private var colorPickerView: some View {
+        VStack {
+            ColorPicker("", selection: $climateViewModel.color)
+        }
+        .frame(width: 45)
+        .padding(.top, -10)
+    }
+    
+    private var stepperView: some View {
+        CustomStepper(
+            value: Binding(
+                get: {
+                    Int(climateViewModel.circleProgress)
+                },
+                set: { newValue in
+                    if 0...15 ~= newValue {
```

Comment:
```suggestion
                    if 0...15 ~= newValue {
```
это не должно быть во вьюмодели?